react/README.md

```diff
@@ -740,7 +740,7 @@ We don’t recommend using indexes for keys if the order of items may change.
 
   This JSX/React style guide is also available in other languages:
 
-  - ![cn](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/China.png) **Chinese (Simplified)**: [jhcccc/javascript](https://github.com/jhcccc/javascript/tree/master/react)
```

Comment:
this redirects just fine for me already. does it not for you?

Reply:
Oh! At first it didn't work for me. But now it does. Seems like it took Github a few minutes to figure out redirection. Thanks you and I'll close this PR